daprdocs/content/en/reference/components-reference/supported-pubsub/setup-jetstream.md

```diff
@@ -82,8 +82,32 @@ helm repo add nats https://nats-io.github.io/k8s/helm/charts/
 helm install my-nats nats/nats
 ```
 
-This installs a single NATS server into the `default` namespace. To interact
-with NATS, find the service with: `kubectl get svc my-nats`.
+This installs a single NATS server into the `default` namespace. To interact with NATS, find the service with: `kubectl get svc my-nats`.
+
+Run `docker ps` and verify `my-nats` is running:
```

Comment:
This seems weird. Helm install applies to Kubernetes anywhere. The command to check that things are running would be `kubectl get pods` or `kubectl get deployments`.